### ISSUE TITLE
The ITM and LM loss of BLIP do not converge

### ISSUE THREAD

**Issue by chenyzh28** [salesforce/LAVIS#282]
Thanks for your work! But I have met some problems when I pretrain BLIP on a chinese dataset (about 4M, coco + vg + some part of cc3m and cc12m). I found that the ITA loss can converge after 20 epochs, but the ITM and LM loss are still high. 
![企业微信截图_2718cf3e-622a-42ad-9200-d37b10f8ed0d](https://user-images.githubusercontent.com/35193979/237017620-a363cfae-c0cf-41a7-95d5-37c1f304bab3.png)
I also test the model performance on COCO validation set (chinese), and it seems normal if I only calculate the ITC similarity.
![企业微信截图_6a7dce82-3bd3-48e8-a96a-575d45aa7e32](https://user-images.githubusercontent.com/35193979/237018585-2c7fb782-8c0e-45f9-a292-23793b277f3e.png)
But when I use the ITM score, since the model does not converge on ITM and LM tasks, the evaluation performance is poor. I notice that `score_matrix_i2t[start+i,topk_idx] = score + topk_sim`, so the TR benefits from ITC task and nothing from ITM task. 
![企业微信截图_56b48076-961e-41d9-b266-315ba20958df](https://user-images.githubusercontent.com/35193979/237018921-7ea9f48f-5d31-4244-946b-2ef59d8f9b7c.png)
Do you know where the problem might be? It would be greatly appreciated if any advice is given.
When I pretrain on chinese dataset, I translate the caption to chinese, and initialize the tokenizer and text encoder with "bert-base-chinese", no other modules have been modified.
![企业微信截图_80f5bf4b-4a66-45de-bea2-7a688da969e1](https://user-images.githubusercontent.com/35193979/237020273-7810b27f-d1a9-42bd-86f5-12425bc0c195.png)

**Comment by LiJunnan1992** [comment]
Can you successfully train with English data? If training with English data has no problem, than I suspect there might be some difference between the English BERT and the Chinese BERT that causes your issue.

**Comment by chenyzh28** [comment]
Thanks for your reply. It seems that the difference between Chinese and English BERT did cause this problem. I lowered the learning rate and ITM and LM are currently converging normally.

**Comment by SinghiCloud** [comment]
> Thanks for your reply. It seems that the difference between Chinese and English BERT did cause this problem. I lowered the learning rate and ITM and LM are currently converging normally.

hi, I meet the same question now, could you give me some advice about how to change thr lr?

**Comment by ldfandian** [comment]
> Thanks for your reply. It seems that the difference between Chinese and English BERT did cause this problem. I lowered the learning rate and ITM and LM are currently converging normally.

Just curios, do you change from "bert-base-uncased" to "bert-base-multilingual-cased" for tokenizer(BertTokenizer) and Qformer(BertLMHeadModel) ? @chenyzh28

**Comment by chenyzh28** [comment]
@ldfandian , Sorry for the late reply. I use the "bert-base-chinese" model as the text encoder of BLIP. Q-former is proposed in BLIP-2, which I have not trained.

**Comment by thesby** [comment]
> Thanks for your reply. It seems that the difference between Chinese and English BERT did cause this problem. I lowered the learning rate and ITM and LM are currently converging normally.

Same problem, how to use bert-base-chinese when pretraining?

**Comment by bfan1256** [comment]
Unfortunately, I am also getting this result, learning rate has not made any change to it.
